x/incentive/keeper/keeper.go

```diff
@@ -0,0 +1,203 @@
+package keeper
+
+import (
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/params/subspace"
+	"github.com/kava-labs/kava/x/incentive/types"
+)
+
+// Keeper keeper for the incentive module
+type Keeper struct {
+	key           sdk.StoreKey
+	cdc           *codec.Codec
+	paramSubspace subspace.Subspace
+	supplyKeeper  types.SupplyKeeper
+	cdpKeeper     types.CdpKeeper
+	accountKeeper types.AccountKeeper
+	codespace     sdk.CodespaceType
+}
+
+// NewKeeper creates a new keeper
+func NewKeeper(cdc *codec.Codec, key sdk.StoreKey, paramstore subspace.Subspace, sk types.SupplyKeeper, cdpk types.CdpKeeper, ak types.AccountKeeper, codespace sdk.CodespaceType) Keeper {
```

Comment:
nit break function signature into multiple lines. I personally use the following:

```go
func Func(
    param1, ..., paramN,
)
```